stl/inc/any

```diff
@@ -357,11 +357,13 @@ inline void swap(any& _Left, any& _Right) noexcept {
     _Left.swap(_Right);
 }
 
-template <class _ValueType, class... _Types>
+template <class _ValueType, class... _Types,
+    enable_if_t<is_constructible_v<any, in_place_type_t<_ValueType>, _Types...>, int> = 0>

```

Comment:
No change requested: I observe that `in_place_type_t<_ValueType>` doesn't match the constness and value category of the function argument, but it doesn't matter (we control the overload set enough). @CaseyCarter agrees that the shorter form is better here.